Dockerfile

```diff
@@ -1,12 +1,20 @@
 FROM ubuntu:18.04 AS build
-WORKDIR /go/src/github.com/prebid/prebid-server/
 RUN apt-get update && \
     apt-get -y upgrade && \
-    apt-get install -y git golang go-dep && \
+    apt-get install -y wget
+RUN cd /tmp && \
+    wget https://dl.google.com/go/go1.11.11.linux-amd64.tar.gz && \
+    tar -xf go1.11.11.linux-amd64.tar.gz && \
+    mv go /usr/local
+WORKDIR /go/src/github.com/prebid/prebid-server/
+ENV GOROOT=/usr/local/go
+ENV GOPATH=$HOME/go
+ENV PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+RUN apt-get install -y git go-dep && \
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-ENV GOPATH /go
 ENV CGO_ENABLED 0
 COPY ./ ./
+RUN echo $(go version)
```

Comment:
probably can remove this line.

Reply:
Ah! Good catch. That's a leftover. I will update it